### PR TITLE
Run3-Trk07 Take care of unit test failure in SimTracker/TrackerMaterialAnalysis

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Take care of unit test failure in SimTracker/TrackerMaterialAnalysis

#### PR validation:

Ran the scripts in the test area

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@qliphy @silviodonato Please do the test once the issue of PPS xml files is resolved

**Comment by perrotta** [comment]
please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
    from Configuration.AlCa.GlobalTag import GlobalTag
    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
 else:
-   process.load('Configuration.Geometry.GeometryExtended2026D49_cff')
+   process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
```
Comment (mmusich) [inline SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py:55]: Why this requires the reco geometry?

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py (hunk 1 above)
Comment (bsunanda) [reply SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py:55]: The reason is HCAL went through a phase where HE was considered a possibility for Phase2 detector. That needs the grouping at SIM and RECO level to be different. Later Reco regrouping was needed in some SIM related studies. That is why it is always better to include the so called RECO geometry which any way has the SIM geometry inside it.

**Comment by bsunanda** [comment]
@cvuosalo @qliphy @perrotta @silviodonato @civanch Kindly approve and integrate this PR on an urgent basis to remove the unit test failure message

**Comment by cvuosalo** [comment]
+geometry

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**